src/core/utils/popup-auto-open.js

```diff
@@ -18,9 +18,9 @@ const openOnScroll = (popup, scrollThreshold) => {
     const offsetTop = window.pageYOffset || document.documentElement.scrollTop
     const clientTop = document.documentElement.clientTop || 0
     const scrollTopPixels = offsetTop - clientTop
-    const scrollTopPercentage = scrollTopPixels / document.body.clientHeight * 100
-    const viewportHeight = Math.max(document.documentElement.clientHeight || 0, window.innerHeight || 0)
-    if (scrollTopPercentage >= scrollThreshold || scrollTopPixels + viewportHeight >= document.body.clientHeight) {
+    const scrollTopPercentage = scrollTopPixels / document.documentElement.clientHeight * 100
+
+    if (scrollTopPercentage >= scrollThreshold || scrollTopPixels + window.innerHeight >= document.documentElement.clientHeight) {
```

Comment:
Let's extract this condition into a boolean variable. When such a var has a meaningful name, it would be easier to read and understand the code :)